manager/app/src/main/java/me/weishu/kernelsu/ui/screen/ExecuteModuleAction.kt

```diff
@@ -79,29 +92,43 @@ fun ExecuteModuleActionScreen(navigator: DestinationsNavigator, moduleId: String
                 actionResult = it
             }
         }
-        if (actionResult) navigator.popBackStack()
+        isActionRunning = false
     }
 
     Scaffold(
         topBar = {
             TopBar(
-                onBack = {
-                    navigator.popBackStack()
-                },
+                isActionRunning = isActionRunning,
                 onSave = {
-                    scope.launch {
-                        val format = SimpleDateFormat("yyyy-MM-dd-HH-mm-ss", Locale.getDefault())
-                        val date = format.format(Date())
-                        val file = File(
-                            Environment.getExternalStoragePublicDirectory(Environment.DIRECTORY_DOWNLOADS),
-                            "KernelSU_module_action_log_${date}.log"
-                        )
-                        file.writeText(logContent.toString())
-                        snackBarHost.showSnackbar("Log saved to ${file.absolutePath}")
+                    if (!isActionRunning) {
+                        scope.launch {
+                            val format = SimpleDateFormat("yyyy-MM-dd-HH-mm-ss", Locale.getDefault())
+                            val date = format.format(Date())
+                            val file = File(
+                                Environment.getExternalStoragePublicDirectory(Environment.DIRECTORY_DOWNLOADS),
+                                "KernelSU_Next_module_action_log_${date}.log"
```

Comment:
Remove the `Next` here?

Reply:
Lmao I forgot to get rid of it from my branch xD